app/config/sources.inc.php

```diff
@@ -444,6 +446,8 @@
     'apple_description_release.lang'        => '2016-03-30', // appstores project
     'apple_screenshots_v3.lang'             => '2016-03-25', // appstores project
     'download_button.lang'                  => '2016-04-29',
+    'emails/2016/fundraising_email_1.lang'  => '2016-12-04',
```

Comment:
Yeah :( They want to send it on the 7th, and I asked for test emails to be sent before